kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -751,6 +751,57 @@ object KyuubiConf {
       .booleanConf
       .createWithDefault(true)
 
+  //------------------------Jetty Server-------------------------
+  val FRONTEND_JETTYSERVER_USE_SSL: ConfigEntry[Boolean] =
```

Comment:
after introducing new configurations, perform `dev/gen/gen_all_config_docs.sh` to update the docs.